src/components/WalletDropdown/MiniPortfolio/PortfolioLogo.tsx

```diff
@@ -60,31 +60,20 @@ const ENSAvatarImg = styled.img`
 const StyledChainLogo = styled.img<{ isSquare: boolean }>`
   height: ${({ isSquare }) => (isSquare ? '16px' : '14px')};
   width: ${({ isSquare }) => (isSquare ? '16px' : '14px')};
-  margin-top: ${({ isSquare }) => (isSquare ? '0px' : '1px')};
-  margin-left: ${({ isSquare }) => (isSquare ? '0px' : '1px')};
-  position: absolute;
-  top: 68%;
-  left: 68%;
 `
 
-const ChainLogoSquareBackground = styled.div`
-  height: 18px;
-  width: 18px;
-  border-radius: 4px;
-  background-color: ${({ theme }) => theme.backgroundSurface};
+const L2LogoContainer = styled.div<{ nonSquareLogo?: boolean }>`
```

Comment:
since there is a `isSquare` below, i think i'd prefer to keep the props not negative like squareLogo (or potentially the same name isSquare?) feels like a good general rule imo